src/content/projects/vvd-viewer.md

```diff
@@ -12,15 +12,8 @@ installation instructions link: https://github.com/JaneliaSciComp/VVDViewer?tab=
 how to cite link: https://doi.org/10.5281/zenodo.5841616
 how to cite text: https://doi.org/10.5281/zenodo.5841616
 additional links array:
-   [
-     https://github.com/JaneliaSciComp/VVDViewer/releases/latest,
-     https://github.com/JaneliaSciComp/VVDViewer/wiki
```

Comment:
@takashi310 I deleted the link to https://github.com/JaneliaSciComp/VVDViewer/wiki, since this already appears under the "documentation link" data field (used in the "Quick Links" column on the right hand side of the page). Let me know if you'd prefer to add it back, to have the documentation link appear in two different locations on the page.